runtime/src/registry/mod.rs

```diff
@@ -0,0 +1,264 @@
+//! This substrate pallet defines a Verifiable Attributes Registry
```

Comment:
Should we rename this Pallet, since `registry` is pretty generic. Maybe nft_registry?

Reply:
How about va_registry?